src/character.cpp

```diff
@@ -2001,3 +2001,36 @@ bool Character::pour_into( vehicle &veh, item &liquid )
     }
     return true;
 }
+
+long Character::ammo_count_for( const item &gun )
+{
+    // INT_MAX instead of LONG_MAX because some code may not expect > INT_MAX
```

Comment:
I don't know which one specifically, but I recall there being a bunch of `ints` around.
Java/C# got it right with numeric values requiring explicit casting.